plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js

```diff
@@ -94,28 +84,26 @@ class WzConfigurationIntegrations extends Component {
             />
           )}
         {currentConfig['integrator-integration'] &&
-          !isString(currentConfig['integrator-integration']) ? (
-            integrations &&
-              integrations.map((integrationInfo, key) => {
-                const integration = Object.assign(
-                  this.buildIntegration(integrationInfo.name),
-                  integrationInfo
-                );
-                return (
-                  <Fragment key={`integration-${integration.title}`}>
-                    <WzConfigurationSettingsGroup
-                      title={integration.title}
-                      description={integration.description}
-                      items={integrationsSettings}
-                      config={integration}
-                      help={key === 0 ? helpLinks : undefined}
-                    />
-                  </Fragment>
-                );
-              })
-          )
-          : null
-        }
```

Comment:
**nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.